pulsar-client/src/test/java/org/apache/pulsar/client/impl/TypedMessageBuilderImplTest.java

```diff
@@ -225,5 +226,12 @@ public void testSetKeyBytesEncodingTypeSeparated() {
         }
     }
 
+    @Test
+    public void test() {
+        producerBase = mock(ProducerBase.class);
+        TypedMessageBuilderImpl builder =  new TypedMessageBuilderImpl(producerBase, null);
+        AutoConsumeSchema autoConsumeSchema = new AutoConsumeSchema();
```

Comment:
I do not understand this test.

what is it supposed to test ? autoConsumeSchema is not used

Reply:
The method `message.getValue()` will use the AutoConsumeSchema to decode data.
Because the schema version is null, so fallback uses the BYTES schema to decode data.

Reply:
Oh, I forgot to remove this, thanks.